README.md

```diff
@@ -42,8 +42,9 @@ By default, configuration is loaded from `tslint.json`, if it exists in the curr
 tslint accepts the following commandline options:
 
     -f, --file:
-        The location of the TypeScript file that you wish to lint. This
-        option is required. Muliptle files are processed consecutively.
+        The location of the TypeScript file that you wish to lint. Multiple
+        files are processed consecutively. Files can also be tacked on to the
+        end of the command.
```

Comment:
hmm you're right, that's a fairly breaking change.

@ashwinr @gscshoyru if we do decide to remove the `-f` option, should we bump the major version? Or is a minor version bump sufficient?

Until then, let's add some sort of deprecation notice for this option (I guess most of my other comments don't apply anymore)